tests/unit_tests/dynamics/collisions/test_sdm_breakup.py

```diff
@@ -320,22 +320,22 @@ def test_attribute_update_single_breakup(
 
         # Assert
         {
-            "multiplicity": lambda: np.testing.assert_array_equal(
+            "multiplicity": lambda: np.testing.assert_array_almost_equal(
                 particulator.attributes["multiplicity"].to_ndarray(),
```

Comment:
multiplicity is `int`, so we should check if it is strictly equal

Reply:
but it fails with equal :(